lib/rspec/mocks/method_double.rb

```diff
@@ -249,6 +249,11 @@ def new_rspec_prepended_module
         end
       end
 
+      def method_owner
+        @method_owner ||=
+          Object.instance_method(:method).bind(object).call(@method_name).owner
```

Comment:
It took me a while to figure out why we have to use `Object#method` instead of using `object.method` directly. For the future reference: it is to tolerate `method` method from being overridden, like here:
```ruby
let(:request_klass) { Struct.new(:method, :url) }
```

Reply:
Yes, I added a comment.